tests/cli/add_database_test.py

```diff
@@ -0,0 +1,79 @@
+
+
+from pathlib import Path
+import tempfile
+
+import pytest
+import yaml
+
+from datajunction.cli import add_database
+from datajunction.errors import DJException
+
+@pytest.mark.asyncio
+async def test_add_database_minimum_requirements() -> None:
+    """
+    Test adding a database with the minimum required arguments
+    """
+    with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
It's better to use the `fs` fixture, it creates a fake filesystem and then you can create any directories you want. See an example here:

https://github.com/DataJunction/datajunction/blob/e90897cf7b733aef682cee5d2a59b1e94c2fa714/tests/cli/compile_test.py#L42

Using `fs` should be faster (because it simulates the disk in memory) and is usually less troublesome to run in the CI/CD IMHO.

Reply:
Updated to use this and it worked great. Thanks!